src/components/Translatathon/DatesAndTimeline.tsx

```diff
@@ -62,58 +61,49 @@ export const DatesAndTimeline = () => {
   const todaysDate = new Date()
 
   return (
-    <Flex direction="column" p={4} mb={16}>
+    <Flex className="mb-16 flex flex-col p-4">
       {dates.map((date, index) => {
         const isLive =
           todaysDate >= date.startDate &&
           (date.endDate ? todaysDate <= date.endDate : true)
         return (
           <Flex
             key={index}
-            borderLeft={"1px solid"}
-            borderColor={
-              index === dates.length - 1 ? "transparent" : "primary.base"
-            }
-            px={4}
-            pb={index === dates.length - 1 ? 0 : 16}
-            gap={4}
+            className={`flex border-l ${index === dates.length - 1 ? "border-transparent" : "border-purple-600 dark:border-purple-400"} px-4 ${index === dates.length - 1 ? "pb-0" : "pb-16"} gap-4`}
```

Comment:
Let's take advantage of the `cn` function for conditional styles. https://github.com/ethereum/ethereum-org-website/blob/dev/src/lib/utils/cn.ts

In this case we could have something like this
```tsx
className={cn("flex border-l", index === dates.length - 1 ? "border-transparent" : "border-primary", ....)}
```

Also, use `border-primary` for the purple color. This is a color from the DS and is already included. You can check them out in this line https://github.com/0xgun/ethereum-org-website/blob/dev/tailwind.config.ts#L164